src/NuGet.Core/NuGet.Commands/LocalsCommandRunner.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Common;
+using NuGet.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+namespace NuGet.Commands
+{
+    public class LocalsCommandRunner
+    {
+        private enum LocalResourceName
+        {
+            Unknown,
+            HttpCache,
+            GlobalPackagesFolder,
+            Temp,
+            All
+        }
+        /// <summary>
+        /// Enum used to indicate the result of the command to callers.
+        /// </summary>
+        public enum LocalsCommandResult
+        {
+            InvalidLocalResourceName,
+            ClearFailure,
+            ClearSuccess
+        }
+        private const string _httpCacheResourceName = "http-cache";
```

Comment:
Picked this up from existing code in Nuget.clients.Localscommand.cs and refactored it as is.
Should I alter this then?